### PR TITLE
update document links

### PR DESCRIPTION
**Pull request by TiansuYu**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
List any dependencies that are required for this change.

Fixes # issue

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Refactoring including code style reformatting 
- [ ] Other (please describe):

# Checklist:

- [ ] I have read the [contributing guideline doc](https://squirrel-core.readthedocs.io/en/latest/usage/code_of_conduct.html) (external contributors only)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have kept the PR small so that it can be easily reviewed  
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] All dependency changes have been reflected in the pip requirement files.

### CODE REVIEW

File: docs/usage/contribute.rst [hunk 1]
```diff
@@ -19,4 +19,4 @@ Tests
 -------------------------------------
 
 You can run tests by executing ``pytest``. Prior make sure that you installed the testing extras e.g. via
-``pip install -e .[dev]`` or ``pip install -e '.[dev, zarr]`` if zarr is needed.
+``pip install -e .[dev]`` or ``pip install -e '.[dev, zarr]'`` if zarr is needed.
```
Comment (AlpAribal) [inline docs/usage/contribute.rst:22]: Currently, `pip install -e '.[dev,zarr,torch,dask,gcp]'` is needed to be able to run `pytest` without any args. Also created a ticket about this.
```suggestion
``pip install -e '.[dev,dask,gcp,torch,zarr]'``.
```

**Review by AlpAribal — DISMISSED** [review]
LGTM with minor comments. We can discuss the test-with-flavor issue on the next weekly.

**Comment by TiansuYu** [comment]
Since this is a public repo, I have set any code changes need to be PR reviewed again. Open to suggestions wether we need this or not.

File: docs/usage/contribute.rst [hunk 2]
```diff
@@ -19,4 +19,4 @@ Tests
 -------------------------------------
 
 You can run tests by executing ``pytest``. Prior make sure that you installed the testing extras e.g. via
```
Comment (AlpAribal) [inline docs/usage/contribute.rst:21]:
```suggestion
You can run tests by executing ``pytest``. Prior make sure that you installed the testing extras via
```

**Review by AlpAribal — APPROVED** [review]
LGTM!

**Review by AlpAribal — APPROVED** [review]
LGTM!

**Pull request merged**